lib/libs/opensearch-lib.ts

```diff
@@ -185,13 +180,8 @@ export async function getItem(
   id: string,
 ): Promise<ItemResult | undefined> {
   client = client || (await getClient(host));
-  try {
```

Comment:
removed try/catch so it can be handled up the stack and the ResponseError can be better utilized